### PR TITLE
setup all the necessary libraries for the project

### PR DESCRIPTION
**Pull request by khoa250200**
#3 
setup libraries for the project

### CODE REVIEW

**Review by anhpn0712 — APPROVED** [review]

**Review by hongthamnguyen2k — APPROVED** [review]

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
```
Comment (KhoaNDAHopee) [inline app/assets/stylesheets/application.scss]: Check rubocop, should be have empty line.

File: app/javascript/application.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
 import "controllers"
+// import "@hotwired/turbo-rails"
```
Comment (KhoaNDAHopee) [inline app/javascript/application.js]: Remove if unused

File: config/routes.rb [hunk 3]
```diff
@@ -1,8 +1,5 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
-end
+  mount LetterOpenerWeb::Engine, at: "/letter_opener" if Rails.env.development?
+end
```
Comment (KhoaNDAHopee) [inline config/routes.rb]: Check rubocop, should be have empty line.

File: spec/support/factory_bot.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
```
Comment (KhoaNDAHopee) [inline spec/support/factory_bot.rb]: Check rubocop, should be have empty line.

**Review by KhoaNDAHopee — CHANGES_REQUESTED** [review]
@khoa250200, Please check my comments.

**Review by KhoaNDAHopee — CHANGES_REQUESTED** [review]
@khoa250200 
Add this config for sass-rails
```
# Add to config/application.rb
config.sass.preferred_syntax = :scss
config.sass.line_comments = false
config.sass.cache = false
```

**Review by KhoaNDAHopee — APPROVED** [review]

**Pull request merged**